jerry-core/rcs/rcs-snapshot.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2015 Samsung Electronics Co., Ltd.
+ * Copyright 2015 University of Szeged
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef RCS_SNAPSHOT_H
+#define RCS_SNAPSHOT_H
+
+#include "rcs-cpointer.h"
+
+typedef struct
+{
+  cpointer_t literal_id;
+  uint32_t literal_offset;
+} lit_mem_to_snapshot_id_map_entry_t;
+
+#ifdef JERRY_ENABLE_SNAPSHOT
+extern bool
+rcs_dump_literals_for_snapshot (uint8_t *,
+                                size_t,
+                                size_t *,
+                                lit_mem_to_snapshot_id_map_entry_t **,
+                                uint32_t *,
+                                uint32_t *);
+
+extern bool
+rcs_load_literals_from_snapshot (const uint8_t *,
+                                 uint32_t,
+                                 lit_mem_to_snapshot_id_map_entry_t **,
+                                 uint32_t *);
+#endif /* JERRY_ENABLE_SNAPSHOT */
```

Comment:
It is better to store these functions in `lit` component to keep low-level and high-level logic separate.
`rcs` component was made to support basic operations with records.
`lit` component implements higher-level operations on literals.

So, could you please rename `rcs-snapshot.h/cpp` to `lit-snapshot.h/cpp` and move them to `lit` folder?